.travis.yml

```diff
@@ -1,18 +1,20 @@
-language: python
-before_install:
-  - sudo apt-get update
-
-  # First install libcangjie
-  - sudo apt-get install libsqlite3-dev
-  - git clone git://github.com/Cangjians/libcangjie
-  - cd libcangjie && git checkout 1.x && ./autogen.sh --prefix=/usr && make && sudo make install && cd ..
-  - git clean -dfx
-
-  # Next install pycangjie
-  - sudo apt-get install cython python3-dev
-  - git clone git://github.com/Cangjians/pycangjie
-  - cd pycangjie && git checkout 1.x && ./autogen.sh --prefix=/usr && make && sudo make install && cd ..
-  - git clean -dfx
-
-  - sudo apt-get install autopoint intltool libibus-1.0-dev python3-gobject
-script: ./autogen.sh && make && make check && make distcheck
+language: c
+sudo: required
+branches:
+  only:
+    - master
+services:
+  - docker
+env:
+  - OS=fedora-latest
+  - OS=ubuntu-devel
+  - OS=ubuntu-latest
+  - OS=ubuntu-lts
+install:
+  - sudo apt-get update && sudo apt-get -y install docker-ce
+  - sudo docker pull "cangjians/pycangjie:${OS}"
+  - sudo docker tag "cangjians/pycangjie:${OS}" "cangjians/pycangjie:latest"
+  - sudo docker build --build-arg OS=${OS} -t "cangjians/ibus-cangjie:${OS}" .
```

Comment:
I'd rather we used long options in scripts, as they are inherently more readable.

The advantage of short-form options is they are faster to type, but in a script that's not an actual benefit. 🙂 

As such, please use `--tag` instead of `-t`.